assets/src/scripts/ajax.js

```diff
@@ -17,6 +17,9 @@ export function get(url) {
             else {
                 // Otherwise reject with the status text
                 // which will hopefully be a meaningful error
+                if (window.ga) {
+                    window.ga('send', 'event', 'serviceFailure', req.status, url);
```

Comment:
Should the status be restricted to those in 500s?

Reply:
@jkreft-usgs , what would you like?